x-pack/plugins/encrypted_saved_objects/server/lib/encrypted_saved_objects_client_wrapper.test.ts

```diff
@@ -0,0 +1,622 @@
+/*
```

Comment:
__note:__ I have mixed feelings about these tests, they may be useful, but at the same time they are a bit too hard to read...